cmd/ci-operator-prowgen/main_test.go

```diff
@@ -273,13 +298,171 @@ func TestGenerateJobs(t *testing.T) {
 				},
 			},
 		}, {
+			id: "Promotion.Namespace is 'openshift'",
 			config: &ciop.ReleaseBuildConfiguration{
-				Tests: []ciop.TestStepConfiguration{
-					{As: "images"},
+				Tests: []ciop.TestStepConfiguration{},
+				Images: []ciop.ProjectDirectoryImageBuildStepConfiguration{
+					{},
+				},
+				PromotionConfiguration: &ciop.PromotionConfiguration{
+					Namespace: "openshift",
+				},
+			},
+			org:    "organization",
+			repo:   "repository",
+			branch: "branch",
+			expected: &prowconfig.JobConfig{
+				Presubmits: map[string][]prowconfig.Presubmit{
+					"organization/repository": {
+						{Name: "pull-ci-organization-repository-branch-images"},
+					},
+				},
+				Postsubmits: map[string][]prowconfig.Postsubmit{
+					"organization/repository": {
+						{
+							Name:   "branch-ci-organization-repository-branch-images",
+							Labels: map[string]string{"artifacts": "images"},
+						},
+					},
+				},
+			},
+		}, {
+			id: "Promotion.Namespace is not 'openshift'",
```

Comment:
Use a cause -> effect in your names so it's easier to understand intended behavior:

    "promotion.namespace is not 'openshift' so no artifacts label is added"